perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -0,0 +1,1558 @@
+%module "AI::MXNetCAPI"
```

Comment:
These API might change in the future. Could you generate them on install (say make perl) instead of copy pasting?